src/org/labkey/test/tests/ExpTest.java

```diff
@@ -131,20 +134,16 @@ public void testSteps()
         _ext4Helper.clickExt4Tab("Source");
         clickButton("Save", 0);
         waitForElement(Locator.css(".labkey-status-info").withText("Saved"));
-        clickButton("Edit Metadata");
-        int created_RowIndex = 5;
-        waitForElement(Locator.name("ff_label" + created_RowIndex), WAIT_FOR_JAVASCRIPT);
-
-        PropertiesEditor editor = PropertiesEditor.PropertiesEditor(getDriver()).withTitle("Metadata Properties").find();
-        PropertiesEditor.FieldRow row = editor.selectField("Created");
-        row.setLabel("editedCreated");
-        PropertiesEditor.FieldPropertyDock.FormatTabPane tabPane = row.properties().selectFormatTab();
-        tabPane.setPropertyFormat("ddd MMM dd yyyy");
-        clickButton("Save", 0);
-        waitForText(WAIT_FOR_JAVASCRIPT, "Save successful.");
+        clickButton("Edit Metadata", 6000);
+        DomainDesignerPage designerPage = new DomainDesignerPage(getDriver());
+
+        DomainFieldRow domainRow = designerPage.fieldsPanel().getField("Created");
+        domainRow.setLabel("editedCreated");
+        domainRow.setDateFormat("ddd MMM dd yyyy");
+        designerPage.click(Locator.button("Save"));
```

Comment:
Doesn't DomainDesignerPage have a clickSave() or clickFinish() method for this?

Reply:
It expects the page to navigate to next page and that doesn't happen here.

Reply:
Ah ok, got it.